lua/astrocommunity/fuzzy-finder/snacks-picker/init.lua

```diff
@@ -0,0 +1,106 @@
+return {
```

Comment:
Maybe add `local snacks = require("snacks")`?

Reply:
This wouldn't do anything